tests/utilities/test_cli.py

```diff
@@ -1437,6 +1441,18 @@ def __init__(self, activation: torch.nn.Module = lazy_instance(torch.nn.LeakyReL
         assert cli.model.activation is not model.activation
 
 
+def test_ddpstrategy_lazy_instance_with_find_unused_parameters():
+    with mock.patch("sys.argv", ["any.py"]):
+        cli = LightningCLI(
+            BoringModel,
+            run=False,
+            trainer_defaults={"strategy": lazy_instance(DDPStrategy, find_unused_parameters=False)},
+        )
+
+    assert cli.config.trainer.strategy.init_args.find_unused_parameters is False
+    assert isinstance(cli.config_init.trainer.strategy, DDPStrategy)
```

Comment:
Can you add the same 2 checks but for the actual instance?

Reply:
It was checking the instance. Anyway I have changed the test checking more things.